pkg/commands/options/testdata/config/.ko.yaml

```diff
@@ -1,2 +1,3 @@
 defaultBaseImage: alpine
 defaultPlatforms: all
+env: CGO_ENABLED=0
```

Comment:
Ko already does some weirdness with `CGO_ENABLED` by default, can you make this something more obviously for testing, like `FOO=bar`, and check that this is set in the build env?